ui/analyse/src/ceval/sunsetterProtocol.js

```diff
@@ -87,14 +54,12 @@ module.exports = function(worker, opts) {
     start: function(w) {
       stopped = null;
       work = w;
-      worker.send('reset crazyhouse');
       worker.send('setboard ' + work.initialFen);
-      worker.send('easy');
-      worker.send('force');
+      worker.send('exit');
```

Comment:
Hack to prevent sunsetter from evaluating while the move list is being sent.

If you don't send 'exit', sunsetter starts in force mode as soon as the first move is sent. If it finds its own move quickly and moves, it can get very confused.

'exit' not needed if we used work.currentFen instead of move list, but there are some edge cases you hit if you don't send moves, (like georgvonzimmermann/Sunsetter#11)